File: digital-sofia-ios/Digital Sofia/Network/Models/ErrorResponse.swift
//
//  ErrorResponse.swift
//  Digital Sofia
//
//  Created by Teodora Georgieva on 14.11.23.
//

import Foundation

struct ErrorResponse: Decodable, Error {
    let error: String
    let errorDescription: String?
    
    private enum CodingKeys: String, CodingKey {
        case error, errorDescription = "error_description"
    }
}

File: digital-sofia-android/data/src/main/java/com/digital/sofia/data/models/network/base/ErrorResponse.kt
/**
 * Please follow code style when editing project
 * Please follow principles of clean architecture
 * Created 2023 by Roman Kryvolapov
 **/
package com.digital.sofia.data.models.network.base

import com.google.gson.annotations.SerializedName

data class ErrorResponse (
    val type: String?,
    val message: String?,
    val error: String?,
    @SerializedName("error_description") val errorDescription: String?
)